utils/hare-status

```diff
@@ -229,16 +231,26 @@ def devices(cns: Consul, node_name: str) -> List[Device]:
 
 @repeat_if_fails(max_retries=24)
 def get_cluster_status(cns: Consul) -> Dict[str, Any]:
-    return {
-        'pools': [x for x in sns_pools(cns)],
-        'profiles': [{'fid': x.fid, 'name': x.name, 'pools': x.pools}
-                     for x in profiles(cns)],
-        'filesystem': get_fs_stats(cns),
-        'nodes': [Node(name=h, svcs=processes(cns, h))
-                  for h in node_names(cns)],
-        'devices': [Devices(node_name=h, devs=devices(cns, h))
-                    for h in node_names(cns)]
-    }
+    if devicesRequested:
```

Comment:
The dicts in `if` and `else` clauses differ with one key only. Please think of the ways how to make the code more readable (e.g. why do we need `else`?).

Reply:
Done